schedule/yam/agama/agama_install_full_disk_encryption..yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: agama_installation
```

Comment:
the name and description still reflects old content.
Agama is an Installer, we could drop `install` from the name.